Cargo.lock

```diff
@@ -121,9 +121,9 @@ checksum = "bef38d45163c2f1dde094a7dfd33ccf595c92905c8f8f4fdc18d06fb1037718a"
 
 [[package]]
 name = "bitflags"
-version = "2.3.2"
+version = "2.3.3"
```

Comment:
Operator changes in bitflags look breaking to me, even though they aren't called out as such: https://github.com/bitflags/bitflags/releases/tag/2.3.3.

Unsure whether it makes a difference for us.

Reply:
Should be fine. @ibeckermayer can you confirm?

Reply:
LGTM